dprof.js

```diff
@@ -147,7 +147,17 @@ asyncHook.enable();
 // Print result
 //
 
-process.on('exit', function () {
+if (process.argv.indexOf('--dprof-no-sigint') === -1 ||
+    !process.env.DPROF_NO_SIGINT) {
+  process.on('SIGINT', () => {
+    writeDataFile();
+    process.exit();
```

Comment:
This indicates a successful exit, however when I SIGINT a process I get the exit code `130`. According to http://unix.stackexchange.com/questions/99112/default-exit-code-when-process-is-terminated exit codes `>= 128` indicates the process exited because of a signal.

I don't know if that is the case here, but surely the exit code should be `>= 1`.